sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client.hpp

```diff
@@ -115,13 +115,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      *
      * @remark This operation requires the certificates/create permission.
      *
-     * @param name The name of the certificate.
      * @param options Options for this operation.
      * @param context The context for the operation can be used for request cancellation.
      * @return CreateCertificateOperation instance used to determine create status.
      */
     CreateCertificateOperation StartCreateCertificate(
-        std::string const& name,
```

Comment:
I might have missed this comment, but why are we removing this?  CreateCert API requires a name for the cert

Reply:
it's in the properties

Reply:
archboard wanted this , to fold it into the options